examples/00-fluent/frozen_rotor_workflow.py

```diff
@@ -0,0 +1,424 @@
+# Define the Viscous Model
+# ==============================================================================================================
+
+# Models setting
+viscous = solver.settings.setup.models.viscous
+viscous.model = "k-omega"
+viscous.k_omega_model = "sst"
+
+################################################################################################################
+# Define Materials
+# ==============================================================================================================
+solver.settings.setup.materials.database.copy_by_name(type="fluid", name="water-liquid")
+
+################################################################################################################
+# Define Cell Zone Conditions
+# ==============================================================================================================
+
+impeller_cell_zone = solver.settings.setup.cell_zone_conditions.fluid["impeller"]
+impeller_cell_zone.general.material = "water-liquid"
+
+impeller_cell_zone.reference_frame.reference_frame_axis_origin = [0, 0, 0]
+impeller_cell_zone.reference_frame.reference_frame_axis_direction = [0, 0, 1]
+impeller_cell_zone.reference_frame.frame_motion = True
+
+# impeller rotation
+impeller_cell_zone.reference_frame.mrf_omega.value = impeller_speed_rad
+
+
+# Volute Cell Zone Conditions
+volute_cell_zone = solver.settings.setup.cell_zone_conditions.fluid["volute"]
+volute_cell_zone.general.material = "water-liquid"
+
+
+# Boundary Conditions
+# impeller hub
+impeller_hub = solver.settings.setup.boundary_conditions.wall["impeller-hub"].momentum
+impeller_hub.wall_motion = "Moving Wall"
+impeller_hub.relative = True
+impeller_hub.velocity_spec = "Rotational"
+
+#  inblock-shroud
+
+inblock_shroud = solver.settings.setup.boundary_conditions.wall[
+    "inblock-shroud"
+].momentum
+inblock_shroud.wall_motion = "Moving Wall"
+inblock_shroud.relative = False
+inblock_shroud.velocity_spec = "Rotational"
+
+################################################################################################################
+# Define Boundary Conditions
+# ==============================================================================================================
+
+# Inlet Boundary Condition
+pressure_inlet = solver.settings.setup.boundary_conditions.pressure_inlet["inlet"]
+pressure_inlet.momentum.supersonic_or_initial_gauge_pressure.value = -100
+
+# It seems, need to change the boundary condition to mass flow outlet
+solver.settings.setup.boundary_conditions.set_zone_type(
+    zone_list=["mass-flow-inlet-11"], new_type="mass-flow-outlet"
+)
+# Outlet Boundary Condition
+
+mass_flow_outlet = solver.settings.setup.boundary_conditions.mass_flow_outlet[
+    "mass-flow-inlet-11"
+]
+mass_flow_outlet.momentum.mass_flow_rate.value = 90  # kg/s
+
+
+# Create a turbo interfaces
+# enable the turbo models
+solver.settings.setup.turbo_models.enabled = True
+
+solver.settings.setup.mesh_interfaces.turbo_interface.create(
```

Comment:
This is giving an issue for me:

>>> solver.settings.setup.mesh_interfaces()
{'interface': {}, 'turbo_create': {}, 'verbosity': 0, 'enforce_continuity_after_bc': False, 'enforce_coupled_wall_between_solids': False, 'enable_visualization_of_interfaces': True, 'mapped_interface_options': {}, 'non_conformal_interface_numerics': {}}


Any idea, why?

Reply:
what's the traceback?

Reply:
Traceback (most recent call last):
  File "d:\Repos\pyfluent\debug.py", line 415, in <module>
    solver.settings.setup.mesh_interfaces.turbo_interface.create(
  File "D:\Repos\pyfluent\src\ansys\fluent\core\solver\flobject.py", line 1145, in __getattribute__
    attr = super().__getattribute__(name)
AttributeError: 'mesh_interfaces' object has no attribute 'turbo_interface'.
The most similar names are: turbo_interface_delete, interface, turbo_interface_health_check.

Reply:
Actually testing the same in @hpohekar's system. Same issue.

Reply:
I have updated it as follows, I have checked with python console it worked as expected. 

```python 

impeller_volute_interface = solver.settings.setup.mesh_interfaces.turbo_create.create(
    adjacent_cell_zone_1="impeller",
    adjacent_cell_zone_2="volute",
    mesh_interface_name="imp-volute-interface",
    turbo_choice="No-Pitch-Scale",
    zone1="interface-impeller-outlet",
    zone2="interface-volute-inlet",

)

```